script.js

```diff
@@ -1,35 +1,253 @@
+  sourceLink: 'https://github.com/DamianDib9/averger',
+},
+
+{
+  picture: 'img/HQproject3.svg',
+  projectName: 'Project 3',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languages: ['Html/CSS', 'Ruby on rails', 'JavaScrip'],
+  seeLink: 'https://damiandib9.github.io/averger/',
+  sourceLink: 'https://github.com/DamianDib9/averger',
+},
+
+{
+  picture: 'img/HQproject4.svg',
+  projectName: 'Project 4',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languages: ['Html/CSS', 'Ruby on rails', 'JavaScrip'],
+  seeLink: 'https://damiandib9.github.io/averger/',
+  sourceLink: 'https://github.com/DamianDib9/averger',
+},
+
+{
+  picture: 'img/HQproject5.svg',
+  projectName: 'Project 5',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languages: ['Html/CSS', 'Ruby on rails', 'JavaScrip'],
+  seeLink: 'https://damiandib9.github.io/averger/',
+  sourceLink: 'https://github.com/DamianDib9/averger',
+},
+
+{
+  picture: 'img/HQproject6.svg',
+  projectName: 'Project 6',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languages: ['Html', 'Ruby on rails', 'JavaScrip', 'CSS'],
+  seeLink: 'https://damiandib9.github.io/averger/',
+  sourceLink: 'https://github.com/DamianDib9/averger',
+},
+];
```

Comment:
- [ ] kindly consider removing the projects section in your html file and create it here as you were doing tho unfortunately is not being loaded on the page  you can check on this [example](https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636) on how to do so

Reply:
Hi @divinecharlotte  I'm sorry but this array is stored on the script.js file. It only goes to the html when the function in triggered by pressing the "see this project" button.
Also I wanted to ask you about this changes on the project.
Are these optional? Because  I already made those changes you asked me here but the second reviewer did not approved me 

![image](https://user-images.githubusercontent.com/121461459/226139728-0ecb2ee0-bfeb-40ad-a371-3157768198be.png)


Thanks for your time.

Reply:
hello @DamianDib9 no optional changes always come with an optional tag on them